### ISSUE TITLE
Project categories broken in master

### ISSUE THREAD

**Issue by torian257x** [lektor/lektor#454]
master seems to have broken the categories implementation ( https://www.getlektor.com/docs/guides/categories/  )
it works when I fall back to 3.0.1
I get:

```
Uncaught (in promise) TypeError: Cannot read property 'isActive' of undefined
    at eval (eval at <anonymous> (http://localhost:5000/admin/static/gen/app.js:664:2), <anonymous>:116:26)
    at Array.map (native)
    at Object.render (eval at <anonymous> (http://localhost:5000/admin/static/gen/app.js:664:2), <anonymous>:107:43)
    at eval (eval at <anonymous> (http://localhost:5000/admin/static/gen/vendor.js:889:2), <anonymous>:795:21)
    at measureLifeCyclePerf (eval at <anonymous> (http://localhost:5000/admin/static/gen/vendor.js:889:2), <anonymous>:75:12)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (eval at <anonymous> (http://localhost:5000/admin/static/gen/vendor.js:889:2), <anonymous>:794:25)
    at ReactCompositeComponentWrapper._renderValidatedComponent (eval at <anonymous> (http://localhost:5000/admin/static/gen/vendor.js:889:2), <anonymous>:821:32)
    at ReactCompositeComponentWrapper.performInitialMount (eval at <anonymous> (http://localhost:5000/admin/static/gen/vendor.js:889:2), <anonymous>:361:30)
    at ReactCompositeComponentWrapper.mountComponent (eval at <anonymous> (http://localhost:5000/admin/static/gen/vendor.js:889:2), <anonymous>:257:21)
    at Object.mountComponent (eval at <anonymous> (http://localhost:5000/admin/static/gen/vendor.js:527:2), <anonymous>:45:35)

```

so I tried:
7c11ba24d7a7f7b89f48ba1ac1d811c013313ccb (was broken)
951c5b3fb01e2514d77a997d38f49a5440cc8395 as well
then went back to
dc92b88e49be3ba655b6aa6efeff1501b1604651
that worked

you guys should probably have a more non trivial test project

**Comment by nixjdm** [comment]
Confirmed. To reproduce, check out branch `example-project` (4926640d0d3a1e16d6fd836903420e4cfc9217ec), run `make build-js` and then `lektor server`, then go to http://127.0.0.1:5000/admin/root:projects:project-a/edit and view the error in the console.

> you guys should probably have a more non trivial test project

[I agree.](https://github.com/lektor/lektor/issues/435)

**Comment by nixjdm** [comment]
This is fixed in an open pull request, https://github.com/lektor/lektor/pull/451. Hopefully we'll pull that soon.

**Comment by nixjdm** [comment]
Fixed by #451
